### DISCUSSION TITLE
Dll reference plugin errors

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12957] [Q&A]
[webpack-cli] Invalid options object. Dll Reference Plugin has been initialized using an options object that does not match the API schema.
 - options.manifest.content['./node_modules/react-redux/node_modules/hoist-non-react-statics/index.js'] has an unknown property 'meta'. These properties are valid:
   object { id, buildMeta?, exports? }
   -> Module info.

I am migrating my project to webpack 5 and not sure why do I receive this and what the fix is. Please help.

**Comment by alexander-akait** [comment]
Change `meta` on `buildMeta`
